sdk/clientcore/tools/pom.xml

```diff
@@ -0,0 +1,14 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>io.clientcore</groupId>
+  <artifactId>clientcore-tools-service</artifactId>
```

Comment:
`clientcore-tools-parent` or something might be more appropriate - but more generally, do we need to have a parent for tools, or can we just put the codegen a level higher?

Reply:
I was just thinking if we add any more tooling specifically for clientcore this would be a good place to club it. I will rename it to `clientcore-tools-parent` for now (easy to take it away if we dont need it later).